res/css/views/settings/tabs/user/_HelpUserSettingsTab.pcss

```diff
@@ -15,6 +15,11 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+/*
+     overflow-x and overflow-y is set to scrollable for 
+     .max_CopyableText class under div tag which is for Access Key
+     field, So it won't go outside boundary.
```

Comment:
this isn't very clear at all. Please have a think about how to make it more succinct.

Reply:
Okay, Got it! Thanks